4-fleetscope/envs/development/versions.tf

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+terraform {
+  required_version = ">= 1.6"
```

Comment:
it is not 1.3 ? what makes 1.6 a requirement?

Reply:
Yeah @yliaog.  For future discussion: `import` blocks were added in v1.5, and the ability to use variables in `import` block ids in v1.6

https://github.com/GoogleCloudPlatform/terraform-google-enterprise-application/blob/main/4-fleetscope/envs/development/main.tf#L21

Reply:
Got it.